topydo/commands/TopCommand.py

```diff
@@ -0,0 +1,105 @@
+# Topydo - A todo.txt client written in Python.
+# Copyright (C) 2014 - 2015 Bram Schoenmakers <me@bramschoenmakers.nl>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from topydo.lib.ExpressionCommand import ExpressionCommand
+from topydo.lib.Config import config
+from topydo.lib.PrettyPrinter import pretty_printer_factory
+from topydo.lib.PrettyPrinterFilter import (
+    PrettyPrinterIndentFilter,
+    PrettyPrinterHideTagFilter,
+    PrettyPrinterBasicPriorityFilter,
+    PrettyPrinterHumanDatesFilter
+)
+from topydo.lib.Utils import get_terminal_size
+
+
+class TopCommand(ExpressionCommand):
```

Comment:
This class has a lot of common with the `ListCommand`. Did you consider to subclass it from `ListCommand` and just override the `_print()` method?

The test cases are also almost identical to the `ListCommand`, so I feel there's some duplication going on here.